src/test/java/tech/jhipster/lite/module/domain/JHipsterModulesFixture.java

```diff
@@ -128,8 +128,10 @@ public static JHipsterModuleBuilder gradleSupportedModuleBuilder() {
         .add(text("Ensure that the input is not null"), "Dummy replacement")
         .and()
       .and()
+    .javaBuildProperties()
+      .set(buildPropertyKey("dummy-dependency.version"), buildPropertyValue("4.5.8"))
```

Comment:
Since build property is usually not used for handling version (this is handled with versionSlug in dependencies and plugins), I think it would be better to use a test example that is not a version:
```suggestion
      .set(buildPropertyKey("spring-profiles-active"), buildPropertyValue("local"))
```

Reply:
Thanks,  it could lead to confusion...👍

Reply:
Everything else looks good! :-)